src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -612,13 +548,13 @@ private void removeTerritoriesThatArentWorthAttacking(final List<ProTerritory> p
         int minCost = Integer.MAX_VALUE;
         Unit minUnit = null;
         for (final Unit u : t.getUnits().getMatches(Matches.unitIsOwnedBy(player))) {
-          if (playerCostMap.getInt(u.getType()) < minCost) {
-            minCost = playerCostMap.getInt(u.getType());
+          if (ProData.unitValueMap.getInt(u.getType()) < minCost) {
```

Comment:
Yeah. I'm still debating on ProData having logic. I think I'm going to try to keep it as just a high level data class that takes current data and player to compute a bunch of standard values that are used. So I think I'll push lots of these little utilities into ProUtilities instead.